parser/src/statements/try_catch_statement.rs

```diff
@@ -1 +1,59 @@
+use std::borrow::Cow;
 
+use crate::{ASTNode, Block, TSXKeyword, TSXToken, VariableIdentifier};
+use source_map::Span;
+use visitable_derive::Visitable;
+
+#[derive(Debug, PartialEq, Eq, Clone, Visitable)]
+#[cfg_attr(feature = "self-rust-tokenize", derive(self_rust_tokenize::SelfRustTokenize))]
+pub struct TryCatchStatement {
+	pub inner: Block, // both try and catch blocks must always be braced
+	pub error_var: VariableIdentifier,
+	pub catch_inner: Block,
```

Comment:
[The spec says catch clause can be missing and there is additionally the finally block](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/try...catch#description). (I couldn't tell you what the `finally` block is used for 😂, need to look when you would use that)
So it should be
```
catch_inner: Option<Block>,
finally_inner: Option<Block>,
```